### DISCUSSION TITLE
[Features] Support Translatable

### DISCUSSION THREAD

**Discussion by mstfkhazaal** [Z3d0X/filament-fabricator#61] [Q&A]
Support title multilanguage via Spatie Laravel Translatable Plugin

https://filamentphp.com/docs/2.x/spatie-laravel-translatable-plugin/installation

**Comment by Voltra** [comment]
You can do so manually for now, by overriding a bunch of stuff. Maybe a complementary package/module would be nice to have all that override out of the box
